packages/synchro-charts/src/components/sc-dial/type.ts

```diff
@@ -0,0 +1,24 @@
+import { Threshold } from '../charts/common/types';
+import { DataPoint, DataStream, MessageOverrides, MinimalSizeConfig, ViewPortConfig } from '../../utils/dataTypes';
+import { LabelsConfig } from '../common/types';
+import { StatusIcon } from '../charts/common/constants';
+
+export type CellOptions = {
```

Comment:
remove cell and grid verbiage, as were removing the grid concept from synchro charts

Reply:
Remove CellOptions